dasharo-compatibility/usb-hid-and-msc-support.robot

```diff
@@ -10,29 +10,32 @@ Library             RequestsLibrary
 # stuff in all test cases
 Resource            ../sonoff-rest-api/sonoff-api.robot
 Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../pikvm-rest-api/pikvm_comm.robot
 Resource            ../variables.robot
 Resource            ../keywords.robot
 Resource            ../keys.robot
+Resource            ../lib/secure-boot-lib.robot
```

Comment:
@filipleple You can add the new libraries to keywords.robot as a resource. Thanks to that you will not need to add them to each test module.

Reply:
and BTW, secure boot library would rather not be required here, if included this way

Reply:
Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)